ubuntu-drivers

```diff
@@ -184,21 +185,33 @@ def command_install(args):
             is_nvidia = True
             break
 
-    # Enable KMS if we are on Wayland
-    if is_nvidia and UbuntuDrivers.detect.is_wayland_session():
+    # Enable KMS if nvidia >= 470
+    for package in to_install:
+        if package.startswith('nvidia-driver-'):
+            try:
+                version = int(package.split('-')[-1])
+            except:
+                pass
+            finally:
+                with_nvidia_kms = version >= 470
+
+    if with_nvidia_kms:
         UbuntuDrivers.detect.set_nvidia_kms(1)
 
+    # --package-list should not install any packages
```

Comment:
This will break networking & storage drivers, as at the moment ubiquity does expect that things do get installed in the live session when called with package-list I think. Since it should be installing things like wifi drivers, ethernet drivers, storage drivers if any.

I think it might make more sense to like skip installing nvidia things, cause well, installing nvidia in live session is never nice.